doc/source/example-a3c.rst

```diff
@@ -147,9 +154,8 @@ global model parameters. The main training script looks like the following.
 Benchmarks and Visualization
 ----------------------------
 
-For the :code:`PongDeterministic-v3` and an Amazon EC2 m4.16xlarge instance, we
-are able to train the agent with 16 workers in around 15 minutes. With 8
```

Comment:
Unsure because the code is the same for the most part -- I'll regress through previous commits today and tomorrow to see if I can find anything